android/src/main/java/io/invertase/firebase/config/RNFirebaseRemoteConfig.java

```diff
@@ -119,9 +120,13 @@ private void fetchInternal(final Promise promise, Boolean withExpiration, long e
         @Override
         public void onComplete(@NonNull Task<Void> task) {
           if (task.isSuccessful()) {
-            promise.resolve("remoteConfigFetchStatusSuccess");
+            promise.resolve("config/success");
```

Comment:
Could change to null - doesn't need to resolve any sort of meaningful value really.